src/clue/adafruit_slideshow.py

```diff
@@ -219,41 +219,43 @@ def update(self):
 
     def __get_next_img(self):
 
-        # handle empty queue
-        if not len(self.pic_queue):
-            if self.loop:
+        if self.direction == PlayBackDirection.FORWARD:
+            self._current_image_index +=1
+            
+            if (self._current_image_index>= len(self.dir_imgs)):

```

Comment:
_&ast;nit&ast;_
Spacing around operators.

Reply:
these will def be fixed after formatting is ran